lib/fluent/plugin/in_forward.rb

```diff
@@ -82,11 +83,15 @@ def shutdown
       @usock.close
       @thread.join
       @lsock.close
```

Comment:
By the way, I have another question about `#shutdown`. Should watchers be detached after `{@usock,@lsock}.close` in `#shutdown`? Otherwise, requests sent from clients between `@loop.watchers.each { |w| w.detach if w.attached? }` and `@lsock.close` would be lost.